src/engraving/layout/layoutsystem.cpp

```diff
@@ -381,8 +422,8 @@ System* LayoutSystem::collectSystem(const LayoutOptions& options, LayoutContext&
             mb->setPos(pos);
             Measure* m = toMeasure(mb);
             qreal stretch = m->basicStretch();
```

Comment:
This one is now no longer needed (nor used) and leads to a compiler warning (in MSVC)